app.js

```diff
@@ -0,0 +1,24 @@
+const query_commands = require('./database/database.js');
+const express = require('express')
+const app = express()
+const port = process.env.PORT || 3000 
+
+const jobObject = {}
+
+app.get('/', (request, response) => {
+    query_commands.getAllJobs()
+        .then( results => {
+            let count = 1; 
+            results.forEach(result => {
```

Comment:
This totally works. And: check out [map](https://www.google.com/search?q=map+mdn&rlz=1C5CHFA_enUS701US701&oq=map+mdn&aqs=chrome..69i57j0l5.1607j0j7&sourceid=chrome&ie=UTF-8) in the future, a little cleaner.